### PR TITLE
[Chore](gensrc) remove gen_vector_functions.py

### PR DESCRIPTION
**Pull request by BiteTheDDDDt**
## Proposed changes

remove unused code generate gen_vector_functions.py

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Comment by BiteTheDDDDt** [comment]
run buildall

File: gensrc/script/Makefile [hunk 1]
```diff
@@ -36,15 +36,6 @@ ${GEN_FUNC_OUTPUT}: gen_functions.py | ${BUILD_DIR}/python
 gen_func: ${GEN_FUNC_OUTPUT}
 .PHONY: gen_func
 
-# generated vector function
-GEN_VEC_FUNC_OUTPUT = ${BUILD_DIR}/python/generated_vector_functions.py \
```
Comment (carlvinhust2012) [inline gensrc/script/Makefile:40]: so will the 'generated_vector_functions.py' not be used again?

Comment (BiteTheDDDDt) [reply gensrc/script/Makefile:40]: > so will the 'generated_vector_functions.py' not be used again?

yes

**Review by yiguolei — APPROVED** [review]
LGTM

**Pull request merged**